client/app/libs/slim_lodash.js

```diff
@@ -0,0 +1,3 @@
+import bindAll from 'lodash/bindAll';
+import find from 'lodash/find';
+export default { bindAll, find };
```

Comment:
The issue is that babel-plugin-lodash does not recognize/support export specifiers; I don't plan to support it either, when using the babel plugin you should use lodash as normal and let the plugin handle the grunt work of importing the modules directly. If you used the methods directly rather than building the slim_lodash file, babel-plugin-lodash would work as expected